learn/framework/index.md

```diff
@@ -1,9 +1,23 @@
 ---
 title: Appchain Development Framework
 description: Learn about the appchain Development Framework, Rust Programming Language, Substrate and its features, including XCM and ready-to-use pallets for your Runtime.
-template: subsection-index-page.html
-hide: 
- - toc
- - feedback
+icon: octicons-code-24
+template: index-page.html
 ---
 
+# Appchain Development Framework
+
+Building a blockchain from scratch is an intricate process that demands deep knowledge of consensus algorithms, cryptography, data structures, governance, and more. Tanssi simplifies this process by leveraging Substrate, a powerful blockchain development framework that provides all the essential components for creating customized blockchains.
+
+With Substrate’s modular architecture, you can modify runtime characteristics, integrate pre-built modules, or develop new ones based on your appchain’s specific needs. Tanssi provides the foundation you need to bring your blockchain vision to life.
+
+Key Features of the Framework:
+
+- **Substrate core architecture** - divided into the *Core Client*, which manages external communications and internal responsibilities like storage, and the *Runtime*, which executes transactions and manages state transitions with customizable logic
```

Comment:
So, this one is kind of weird because there are multiple ways to add bold and italics styling, but we try to stick to the following conventions:

Bold - use `**something in bold**` (instead of `__something in bold__`)
Italics - use `_something in italics_` (instead of `*something in italics*`)

Reply:
Also, we don't need to capitalize Core Client or Runtime here